launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/DataServlet.java

```diff
@@ -14,18 +14,36 @@
 
 package com.google.launchpod.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. */
-@WebServlet("/data")
+@WebServlet("/load-result")
```

Comment:
we should make this endpoint as descriptive as possible. as the project grows "load-results" may be confusing since we may have a lot of endpoints that all load different types of results

Reply:
done.